tests/phpunit/api/v3/NavigationTest.php

```diff
@@ -46,7 +46,7 @@ public function testGet() {
    * Test create function.
    */
   public function testCreate() {
-    $params = array('label' => 'Feed the Goats', 'domain_id' => 1);
+    $params = array('name' => 'feed', 'label' => 'Feed the Goats', 'domain_id' => 1);
```

Comment:
Looking at this suggests it can take name or label https://github.com/civicrm/civicrm-core/blob/master/CRM/Core/BAO/Navigation.php#L104